modules/classic_ui/pages/files/webgui/sharing.adoc

```diff
@@ -133,90 +96,74 @@ This feature may seem a little strange; however, it provides flexibility for all
-In that case, you must set an expiration date less than or equal to the maximum.
+The administrator may have enforced the default expiration to be the maximum expiration. In that case, you must set an expiration date less than or equal to the maximum.
 
-The share will expire at the end of the specified expiration date. Users of the share will
-no longer be able to access it.
+The share will expire at the end of the specified expiration date. Users of the share will no longer be able to access it.
 
 == Creating or Connecting to Federation Share Links
 
-Federated Cloud Sharing allows you to mount file shares from remote
-ownCloud servers, and manage them just like a local share. In ownCloud 8
-the process for creating a new sharing link is easier and more
-streamlined. See xref:files/federated_cloud_sharing.adoc[Using Federation Shares]
-to learn to how to create and connect to new Federated Cloud shares.
+Federated Cloud Sharing allows you to mount file shares from remote ownCloud servers, and manage them just like a local share. In ownCloud 8 the process for creating a new sharing link is easier and more streamlined. See xref:files/federated_cloud_sharing.adoc[Using Federation Shares] to learn to how to create and connect to new Federated Cloud shares.
 
 == Share Permissions
 
 Shares can have a combination of the following five permission types:
 
 [cols="15%,70%",options="header",]
 |===
-| Permission | Definition
-| can share  | Allows the users you share with to re-share
-| can edit   | Allows the users you share with to edit your shared files, and to collaborate using the Documents app
-| create     | Allows the users you share with to create new files and add them to the share
-| change     | Allows uploading a new version of a shared file and replacing it
-| delete     | Allows the users you share with to delete shared files
+| Permission
+| Definition
+
+| can share
+| Allows the users you share with to re-share
+
+| can edit
+| Allows the users you share with to edit your shared files, and to collaborate using the Documents app
+
+| create
+| Allows the users you share with to create new files and add them to the share
+
+| change
+| Allows uploading a new version of a shared file and replacing it
+
+| delete
+| Allows the users you share with to delete shared files
 |===
 
 == Creating Drop Folders
 
-As of ownCloud version 10.0.2, users can create upload-only, public
-shares (otherwise known as "Drop Folders"). Drop Folders allow users
-to upload files to a central location, but don’t allow them to either
-see or change any existing files, which already have been uploaded.
+As of ownCloud version 10.0.2, users can create upload-only, public shares (otherwise known as "Drop Folders"). Drop Folders allow users to upload files to a central location, but don’t allow them to either see or change any existing files, which already have been uploaded.
```

Comment:
```suggestion
As of ownCloud version 10.0.2, users can create upload-only, public shares (otherwise known as "Drop Folders"). Drop Folders allow users to upload files to a central location, but don’t allow them to either see or change any existing files, which have already been uploaded.
```